cmd/ingest.go

```diff
@@ -65,9 +70,22 @@ func (c *ingestCmd) Command() *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:               "ingest",
-		Short:             "Run Ingestion service",
-		PersistentPreRunE: utils.DefaultPersistentPreRunE(cfgOpts),
+		Use:   "ingest",
+		Short: "Run Ingestion service",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			if err := cfgOpts.RequireE(); err != nil {
+				return fmt.Errorf("requiring values of config options: %w", err)
+			}
+			if err := cfgOpts.SetValues(); err != nil {
+				return fmt.Errorf("setting values of config options: %w", err)
+			}
+			appTracker, err := sentry.NewSentryTracker(sentryDSN, stellarEnvironment, 5)
+			if err != nil {
+				log.Fatalf("Error initializing App Tracker: %s", err.Error())
+			}
+			cfg.AppTracker = *appTracker
```

Comment:
Here you should pass the reference because it is an interface!

Reply:
I made it
cfg.AppTracker = appTracker